editor/src/editor/nodes/camera.ts

```diff
@@ -10,6 +10,15 @@ export class EditorCamera extends FreeCamera {
 	private _keyboardUpListener: (ev: KeyboardEvent) => void;
 	private _keyboardDownListener: (ev: KeyboardEvent) => void;
 
+	private static readonly DEFAULT_KEYS = {
```

Comment:
Could be a const above the class, to follow the code style of other classes in the project